File: src/main/kotlin/dev/loveeev/astraTowny/events/response/ResponseSuccessEvent.java
package dev.loveeev.astraTowny.events.response;

import lombok.Getter;
import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;

@Getter
public class ResponseSuccessEvent extends Event {

    private static final HandlerList handlers = new HandlerList();
    private final String message;

    public ResponseSuccessEvent(String message) {
        this.message = message;
    }

    @Override
    public HandlerList getHandlers() {
        return handlers;
    }

    public static HandlerList getHandlerList() {
        return handlers;
    }
}

File: src/main/kotlin/dev/loveeev/astraTowny/events/response/ResponseSuccessEvent.kt
package dev.loveeev.astratowny.events.response

import org.bukkit.event.Event
import org.bukkit.event.HandlerList


class ResponseSuccessEvent(val message: String) : Event() {
    companion object {
        // Статический метод getHandlerList
        @JvmStatic
        fun getHandlerList(): HandlerList {
            return HANDLER_LIST
        }

        // Статическая переменная HandlerList
        val HANDLER_LIST = HandlerList()
    }

    override fun getHandlers(): HandlerList {
        return HANDLER_LIST
    }

}